style.css

```diff
@@ -0,0 +1,839 @@
+  }
+
+  .header-content {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    flex-wrap: wrap;
+    align-items: flex-start;
+    margin: -53rem 0 0 19rem;
+  }
+
+  .header-content .content-title {
+    display: flex;
+    justify-content: flex-start;
+    color: #ec5242;
+    font-size: 40px;
+    font-weight: 400;
+    text-align: center;
+    margin-top: -1rem;
+    margin-left: -23rem;
+  }
+
+  .header-content h1 {
+    font-family: "COCOGOOSE", sans-serif;
+    background-image: url(img/red.png);
+    background-size: cover;
+    background-position: bottom;
+    color: transparent;
+    font-weight: 700;
+    font-size: 60px;
+    background-clip: border-box;
+    -webkit-background-clip: text;
+    margin-left: 3rem;
+    align-items: start;
+  }
+
+  .header-content .header-content-body {
+    background-color: #f7f7f7;
+    border: 0.28rem solid #fff;
+    display: flex;
+    text-align: center;
+    align-items: center;
+    height: auto;
+    padding: 5px;
+    margin-left: 7rem;
+    width: 80%;
+  }
+
+  .header-content-body p {
+    display: flex;
+    font-size: 15px;
+    line-height: 2rem;
+    padding-top: 0.6rem;
+  }
+
+  .header-content .summit-date {
+    color: #555;
+    font-weight: 800;
+    font-size: 30px;
+    padding-top: 1.5rem;
+    padding-left: 6rem;
+  }
+
+  .header-content .summit-venue {
+    color: #555;
+    font-weight: 150;
+    font-size: 20px;
+    padding-top: 0.3rem;
+    text-align: center;
+    margin-left: -4rem;
+  }
+
+  /* Main Program Styles */
+  .main-program {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    background-image: url(img/crosses-bg.png);
+    background-color: #282b32;
+    height: auto;
+    color: #fff;
+    padding-bottom: 70px;
+  }
+
+  .main-program .main-program-title {
+    font-size: 30px;
+    position: relative;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    margin-top: 3rem;
+  }
+
+  .hr-line {
+    color: #ec5242;
+    align-self: center;
+    border-style: solid;
+    border-width: 2px;
+    width: 25%;
+    margin-top: -0.25rem;
+  }
+
+  .programs-card-container {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    flex-direction: row;
+    margin: 50px 200px;
+    background-color: #272a31;
+    gap: 1rem;
+    height: 50%;
+  }
+
+  .program {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    width: 12rem;
+    color: #fff !important;
+  }
+
+  .program .program-title {
+    color: #ec5242;
+    font-weight: 700;
+    font-size: 1.063rem;
+  }
+
+  .program:hover {
+    border: 0.25rem solid #fff;
+  }
+
+  .see-the-whole-program p {
+    font-size: 30px;
+  }
+
+  /* \\\\\\\\Featured Section//////// */
+  #featured-speakers {
+    display: flex;
+    flex-wrap: wrap;
+    position: relative;
+    padding-left: 19rem;
+    padding-right: 19rem;
+    height: 50rem;
+  }
```

Comment:
- [ ] By implementing the suggestion by the previous reviewer, There is a new bug got introduced in your design that has made your whole design look very unprofessional and unresponsive. 
right now your featured speakers section got more width than the other sections of the web-page.
kindly consider fixing this bug in your design
your design

<img width="896" alt="Screenshot 2023-07-08 at 10 46 57 PM" src="https://github.com/Edgahkipkemoi/capstone/assets/96683843/a8cb4034-9442-47b7-b474-d41405ca5f30">